wazo_calld/plugin_helpers/confd.py

```diff
@@ -132,7 +132,10 @@ def from_id(cls, conference_id, confd_client):
 
 def get_user_voicemail(user_uuid, confd_client):
     try:
-        return confd_client.users.get(user_uuid)['voicemail']
+        voicemail = confd_client.users.get(user_uuid)['voicemail']
+        if not voicemail:
+            raise NoSuchUserVoicemail(user_uuid)
+        return voicemail
     except IndexError:
```

Comment:
Is the `IndexError` still useful here? I feel that it was meant to catch a user with no `voicemail` field.

Reply:
Probably not. I will remove it. Thanks!